scripts/mkplaces.py

```diff
@@ -0,0 +1,326 @@
+#! /usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import json
+import sys
+from os import environ
+from copy import deepcopy
+from sys import stderr
+
+from scriptutil import get_nodes_to_ea, decode_file, gen_json_data, calc, substi_temp, dump_json
+
+P2020 = environ.get('P2020','power-qoriq-p2020-ds-p')
+MPC5777M = environ.get('MPC5777M',  'power-mpc5777m-evb')
+NVAL = int(environ.get('STUBBORN_MAX_MEASURES', '1024'))
+NO_SEP = bool(environ.get('NO_SEP', ''))
+IGN = environ.get('TRACE_IGN', '').split(':')
+
```

Comment:
These should be mentioned somewhere in the README as they drive the generation of results.
At least the most important ones.